.github/workflows/tests.yml

```diff
@@ -1,10 +1,20 @@
 name: Tests
 
-on: [push]
+on:
+  push:
+    branches:
+      - "**" # all branches
+      - "!master" # except master
+      - "!develop" # except develop
+
+env:
+  # Point tests to run from newest stable chromedriver
+  CHROMEDRIVER_PATH: /usr/local/bin/chromedriver
 
 jobs:
   lint:
     runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
This is only adding a timeout to the lint job specifically, but the PR title/description makes me think you intended to add one to all jobs, no?

(also, 30 minutes is probably excessive - I'd suggest something more like 5m for the lint job and 15m for the browser-based e2e jobs)